test/mnemonic-test.js

```diff
@@ -64,4 +67,127 @@ describe('Mnemonic', function() {
     assert.strictEqual(m2.language, m1.language);
     assert.bufferEqual(m2.toSeed(), m1.toSeed());
   });
+
+  it('should return true for isMnemonic when passed a Mnemonic object', () => {
+    const m1 = new Mnemonic();
+    assert.strictEqual(Mnemonic.isMnemonic(m1), true);
+  });
+
+  it('should return false for isMnemonic when passed a string', () => {
+    const m1 = new Mnemonic();
+    assert.strictEqual(Mnemonic.isMnemonic(m1.getPhrase()), false);
+  });
+
+  it('should return the phrase from toString', () => {
+    const m1 = new Mnemonic();
+    assert.strictEqual(m1.toString(), m1.getPhrase());
+  });
+
+  it('should handle fromRaw correctly', () => {
+    const m1 = new Mnemonic();
+    const m2 = Mnemonic.fromRaw(m1.toRaw());
+    assert.strictEqual(m1.getPhrase(), m2.getPhrase());
+  });
+
+  it('should handle fromJSON correctly', () => {
+    const m1 = new Mnemonic();
+    const m2 = Mnemonic.fromJSON(m1.toJSON());
+    assert.strictEqual(m1.getPhrase(), m2.getPhrase());
+    assert.strictEqual(m1.language, m2.language);
+    assert.strictEqual(m1.bits, m2.bits);
+    assert.bufferEqual(m1.getEntropy(), m2.getEntropy());
+  });
+
+  it('should expect an error from getLanguage() when word is not in any language wordlist', () => {
+    assert.throws(() => {
+        Mnemonic.getLanguage('notaword');
+      },
+      Error,
+      'Unknown word: notaword');
+  });
+
+  it('should handle fromEntropy correctly', () => {
+    const m1 = new Mnemonic();
+    const m2 = Mnemonic.fromEntropy(m1.getEntropy(), LANGUAGE_ENGLISH);
+    assert.strictEqual(m1.getPhrase(), m2.getPhrase());
+    assert.strictEqual(LANGUAGE_ENGLISH, m2.language);
+    assert.strictEqual(m1.bits, m2.bits);
+    assert.bufferEqual(m1.getEntropy(), m2.getEntropy());
+  });
+
+  it('should expect an error from fromPhrase() when phrase contains a word not in the wordlist', () => {
```

Comment:
You can make these titles a bit shorter too. We try to keep all lines (even in the tests) under 80 columns.
Sometimes in the tests we can ignore that rule, but I think we can keep the style simpler here:
e.g....
```suggestion
  it('should throw when phrase contains unknown word () => {
```